bindings/wasm/examples/public/examples.js

```diff
@@ -109,22 +109,25 @@ export const examples = {
       // parameters.
 
       function roundedFrame(edgeLength, radius, circularSegments = 0) {
-        const edge = cylinder(edgeLength, radius, -1, circularSegments);
-        const corner = sphere(radius, circularSegments);
+        const edge =
+            Manifold.cylinder(edgeLength, radius, -1, circularSegments);
+        const corner = Manifold.sphere(radius, circularSegments);
 
-        const edge1 = union(corner, edge).rotate([-90, 0, 0]).translate([
-          -edgeLength / 2, -edgeLength / 2, 0
-        ]);
+        const edge1 =
+            Manifold.union(corner, edge).rotate([-90, 0, 0]).translate([
+              -edgeLength / 2, -edgeLength / 2, 0
+            ]);
 
-        const edge2 = union(
-            union(edge1, edge1.rotate([0, 0, 180])),
+        const edge2 = Manifold.union(
```

Comment:
Maybe we should destructure at the top: `const {union, cylinder} = Manifold;` so we don't have to add so much verbosity?

Reply:
Within the example body (where they will appear to the user when viewing the example), or above? If it was at the top, that pollute the whole namespace for the editor I guess. The shape constructors have no collision between Manifold and CrossSection if there were opened out, but the booleans would. Not sure what the best course is.

Reply:
In the body where they appear to the user. Basically the same pattern I use with `const {vec3} = glMatrix;`.

Reply:
Ok I'll hunt through for places where it looks like it could use a local opening like that.